app/assets/stylesheets/application.css.scss

```diff
@@ -18,3 +18,175 @@
 @import "bootstrap";
 @import "font-awesome-sprockets";
 @import "font-awesome";
+body {
+  font-family: "century gothic", "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Comment:
Properties should be ordered background-color, color, font-family, font-size, line-height